app/helpers/persistent_volume_helper/textual_summary.rb

```diff
@@ -25,6 +25,11 @@ def textual_group_smart_management
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
+  def textual_group_capacity
+    labels = [_("resource"), _("quantity")]
```

Comment:
@zakiva labels should be titleized, "Resource", "Quantity" unless there is a specific reason for them to be all lowercase.